src/server/pps/server/metrics_test.go

```diff
@@ -0,0 +1,82 @@
+//go:build k8s
+
+package server_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/minikubetestenv"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pctx"
+	"github.com/pachyderm/pachyderm/v2/src/internal/require"
+	"github.com/pachyderm/pachyderm/v2/src/internal/testutil"
+)
+
+func TestContainerMetrics(t *testing.T) {
+	ctx, cancel := pctx.WithCancel(pctx.TestContext(t))
+	defer cancel()
+	t.Parallel()
+	c, namespace := minikubetestenv.AcquireCluster(t, minikubetestenv.WithPrometheus)
+
+	docsDir := t.TempDir()
+	tmpDir := t.TempDir()
+	cmd := testutil.PachctlBashCmdCtx(ctx, t, c, `
+# run a pipeline in order to have a container to scrape metrics from
+pachctl create project "video-to-frame-traces"
```

Comment:
What does quoting "video-to-frame-traces" do here?

Reply:
Nothing. Harmless, but also pointless. Deleted.